airflow/migrations/versions/92c57b58940d_add_fab_tables.py

```diff
@@ -0,0 +1,182 @@
+            sa.Column('email', sa.String(length=64), nullable=False),
+            sa.Column('last_login', sa.DATETIME(), nullable=True),
+            sa.Column('login_count', sa.INTEGER(), nullable=True),
+            sa.Column('fail_login_count', sa.INTEGER(), nullable=True),
+            sa.Column('created_on', sa.DATETIME(), nullable=True),
+            sa.Column('changed_on', sa.DATETIME(), nullable=True),
+            sa.Column('created_by_fk', sa.INTEGER(), nullable=True),
+            sa.Column('changed_by_fk', sa.INTEGER(), nullable=True),
+            sa.ForeignKeyConstraint(['changed_by_fk'], ['ab_user.id']),
+            sa.ForeignKeyConstraint(['created_by_fk'], ['ab_user.id']),
+            sa.PrimaryKeyConstraint('id'),
+            sa.UniqueConstraint('email'),
+            sa.UniqueConstraint('username'),
+        )
+
+    if "ab_user_role" not in tables:
+        op.create_table(
+            'ab_user_role',
+            sa.Column('id', sa.INTEGER(), nullable=False, primary_key=True),
+            sa.Column('user_id', sa.INTEGER(), nullable=True),
+            sa.Column('role_id', sa.INTEGER(), nullable=True),
+            sa.ForeignKeyConstraint(
+                ['role_id'],
+                ['ab_role.id'],
+            ),
+            sa.ForeignKeyConstraint(
+                ['user_id'],
+                ['ab_user.id'],
+            ),
+            sa.PrimaryKeyConstraint('id'),
+            sa.UniqueConstraint('user_id', 'role_id'),
+        )
+
+    if "ab_register_user" not in tables:
+        op.create_table(
+            'ab_register_user',
+            sa.Column('id', sa.INTEGER(), nullable=False, primary_key=True),
+            sa.Column('first_name', sa.String(length=64), nullable=False),
+            sa.Column('last_name', sa.String(length=64), nullable=False),
+            sa.Column('username', sa.String(length=64), nullable=False),
+            sa.Column('password', sa.String(length=256), nullable=True),
+            sa.Column('email', sa.String(length=64), nullable=False),
+            sa.Column('registration_date', sa.DATETIME(), nullable=True),
+            sa.Column('registration_hash', sa.String(length=256), nullable=True),
+            sa.PrimaryKeyConstraint('id'),
+            sa.UniqueConstraint('username'),
+        )
+
+
+def downgrade():
+    """Drop FAB Tables"""
+    conn = op.get_bind()
+    inspector = Inspector.from_engine(conn)
+    tables = inspector.get_table_names()
+    fab_tables = [
+        "ab_permission",
+        "ab_view_menu",
+        "ab_role",
+        "ab_permission_view",
+        "ab_permission_view_role",
+        "ab_user",
+        "ab_user_role",
+        "ab_register_user",
+    ]
+
+    print()
+    for table in fab_tables:
+        if table in tables:
+
+            indexes = inspector.get_foreign_keys(table)
+            for index in indexes:
+                index_name = index.get('name')
+                print(f"Dropping Index: {index_name}")
```

Comment:
```suggestion
```
Probably shouldn't have print statements in the migrations?

Reply:
I kept all the print statements intentionally to show the following:

```
Dropping Index: ab_permission_view_ibfk_1
Dropping Index: ab_permission_view_ibfk_2
Dropping Index: ab_permission_view_role_ibfk_1
Dropping Index: ab_permission_view_role_ibfk_2
Dropping Index: ab_user_ibfk_1
Dropping Index: ab_user_ibfk_2
Dropping Index: ab_user_role_ibfk_1
Dropping Index: ab_user_role_ibfk_2

Dropping Table: ab_permission
Dropping Table: ab_view_menu
Dropping Table: ab_role
Dropping Table: ab_permission_view
Dropping Table: ab_permission_view_role
Dropping Table: ab_user
Dropping Table: ab_user_role
Dropping Table: ab_register_user
```

Reply:
NO Strong Opinion here

Reply:
Removed: https://github.com/apache/airflow/pull/12352/commits/1f21b8b20c3cdf33e64efc9de329b5152aeaded1

Reply:
(I think if we want to keep them we should convert them to log.info calls. Up to you though.)